tests/Repository/AamcMethodRepositoryTest.php

```diff
@@ -80,7 +77,7 @@ public function testFindDtosByWithCacheEnabled()
     protected function tearDown(): void
     {
         parent::tearDown();
-
+        unset($this->fixtures);
```

Comment:
🐶! How many of these are we missing I wonder, this is maybe where our tests are leaking so much memory!